Error on generating images with one of my .osd files

Can't generate image files from one of my .osd files (attached in the zip file). It works fine with version 0.5 but any newer release gives error:

```
DEBUG:root:Current: 2800/12570 (fps: 277)
Exception in thread Thread-1 (main):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "processor.py", line 632, in main
  File "processor.py", line 536, in __render_osd_frame
  File "processor.py", line 536, in <listcomp>
cv2.error: OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv\modules\core\src\matrix_operations.cpp:67: error: (-215:Assertion failed) src[i].dims <= 2 && src[i].rows == src[0].rows && src[i].type() == src[0].type() in function 'cv::hconcat'
```
[AvatarG0004.zip](https://github.com/kirek007/ws-osd-py/files/10473123/AvatarG0004.zip)

--- a/processor.py
+++ b/processor.py
@@ -288,7 +288,6 @@ class Utils:
         # Image ranges
         y1, y2 = max(0, y), min(img.shape[0], y + img_overlay_res.shape[0])
         x1, x2 = max(0, x), min(img.shape[1], x + img_overlay_res.shape[1])
-
         # Overlay ranges
         y1o, y2o = max(0, -y), min(img_overlay_res.shape[0], img.shape[0] - y)
         x1o, x2o = max(0, -x), min(img_overlay_res.shape[1], img.shape[1] - x)
@@ -669,7 +668,7 @@ class OsdGenerator:
                 osd_time = raw_osd_frame.startTime
                 Utils.merge_images(frame, osd_frame, self.config.offset_left,
                                    self.config.offset_top, self.config.osd_zoom)
-                osd_frame_no_srt = osd_frame
+                osd_frame_no_srt = frame
                 
             if self.srt and self.config.include_srt:
                 if srt_time < calc_video_time:

